.github/workflows/ci.yml

```diff
@@ -19,45 +19,41 @@ env:
   #   - https://github.com/privacy-scaling-explorations/mpz/issues/178
   # 32 seems to be big enough for the foreseeable future
   RAYON_NUM_THREADS: 32
+  # these remove the need for git in notary-server build.rs:
+  GIT_COMMIT_HASH: ${{ github.event.pull_request.head.sha || github.sha }}
+  GIT_COMMIT_TIMESTAMP: ${{ github.event.repository.updated_at}}
```

Comment:
does `github.event.repository.updated_at` always refer to the timestamp of the head commit? It seems like the timestamp ll also be updated if someone updates e.g. the description of the repo (https://github.com/orgs/community/discussions/24442)

Reply:
i coulnd't find the documentation for `github.event.repository.updated_at`, the link above is the closest i can find — worth double checking imo